cmd/lakefs/cmd/run.go

```diff
@@ -66,6 +66,38 @@ func checkAuthModeSupport(cfg *config.Config) error {
 	return nil
 }
 
+func CreateAuthService(ctx context.Context, cfg *config.Config, logger logging.Logger, kvStore kv.Store) auth.Service {
```

Comment:
I know you've been over this with @itaigilo , but... it's a constructor, and [by convention](https://go.dev/doc/effective_go#composite_literals) should be named NewAuthService.

Reply:
Thanks, changed